yagi/src/main/java/org/raml/yagi/framework/util/DateUtils.java

```diff
@@ -25,52 +25,120 @@
 
 public class DateUtils
 {
-    public static final String DATE_ONLY_FOUR_DIGITS_YEAR_LENGTH_VALIDATION = "yagi.date_only_four_digits_year_length_validation";
-    public static boolean FOUR_YEARS_VALIDATION = Boolean.valueOf(System.getProperty(DATE_ONLY_FOUR_DIGITS_YEAR_LENGTH_VALIDATION, "true"));
+    private static final String DATE_ONLY_FOUR_DIGITS_YEAR_LENGTH_VALIDATION = "yagi.date_only_four_digits_year_length_validation";
+    private static final String STRICT_DATES_RFC3339 = "org.raml.strict_dates_rfc3339";
```

Comment:
Let's keep the same name "convention" as the one above, i.e. `DATE_RFC3339_VALIDATION`